ParabolInc/parabol
# feat(standups): Editable Prompt within meeting

# Description
fixes https://github.com/ParabolInc/parabol/issues/6584
Note that this does _not_ match the styling on mobile.  Due to some styling used as part of the `useModal` hook internals, we'd need to either refactor some styles, or implement a mobile-friendly overlay using lower-level hook APIs.

## Demo
https://www.loom.com/share/7b8aac4235594fc18003a4f26f6c7336

## Testing scenarios
* Create a new meeting as facilitator, and also join as a separate user
* Confirm that there is an initial meeting prompt
* Click the meeting prompt and see a modal open
* Click a suggested prompt and confirm that it populates the text box
* Modify the prompt in the text box
* Click "Use Prompt"
* Confirm that the new prompt appears on the facilitator's view, and the other user's view updated to the new prompt
* As facilitator, end the meeting from the options menu
* Confirm that the prompt no longer appears editable, and the modal does not open when clicking

## Final checklist
